src/pybind/mgr/nfs/tests/test_nfs.py

```diff
@@ -789,8 +789,8 @@ def _do_test_remove_export(self) -> None:
         nfs_mod = Module('nfs', '', '')
         conf = ExportMgr(nfs_mod)
         assert len(conf.exports[self.cluster_id]) == 2
-        assert conf.delete_export(cluster_id=self.cluster_id,
-                                  pseudo_path="/rgw") == (0, "Successfully deleted export", "")
+        conf.delete_export(cluster_id=self.cluster_id,
+                           pseudo_path="/rgw")
```

Comment:
this removes the assert?

Reply:
yes, if the function encounters a problem it will raise an exception now.  it no longer tries to behave like a "mgr endpoint" function

Reply:
was just confirming that you didn't delete it by mistake :)